nikola/__main__.py

```diff
@@ -289,7 +285,7 @@ def load_tasks(self, cmd, opt_values, pos_args):
             signal('initialized').send(self.nikola)
         except Exception:
             LOGGER.error('Error loading tasks')
-            raise
+            sys.exit(3)
```

Comment:
This is terrible: in case something happens, there is no way to see WHAT happened without modifying Nikola's source code! I've defined `NIKOLA_DEBUG`, but I still just get `Error loading tasks`.

I had to undo this change to get some useful output.